src/gc/impl/conservative/gc.d

```diff
@@ -2557,6 +2559,8 @@ Lmark:
                 }
             }
 
+            if(shouldFork)
+                thread_suspendAll();
```

Comment:
Threads are not suspended if it's not forking?

Also, I don't get how this is related to `minimize()`.